doc/api/util.md

```diff
@@ -534,24 +613,40 @@ util.isUndefined(null)
 ```
 
 ## util.log(string)
+<!-- YAML
+added: v0.3.0
+deprecated: v6.0.0
+-->
 
 Output with timestamp on `stdout`.
 
     require('util').log('Timestamped message.');
 
 ## util.print([...])
+<!-- YAML
+added: v0.3.0
+deprecated: v0.11.3
+-->
 
 > Stability: 0 - Deprecated: Use [`console.log()`][] instead.
 
 Deprecated predecessor of `console.log`.
 
 ## util.pump(readableStream, writableStream[, callback])
+<!-- YAML
```

Comment:
This comment is not present in the original commit as `util.pump()` has been removed from the documentation.
Added here for completeness.